src/credman.c

```diff
@@ -249,6 +250,10 @@ credman_parse_rk(const cbor_item_t *key, const cbor_item_t *val, void *arg)
 			return (-1);
 		cred->type = cred->attcred.type; /* XXX */
 		return (0);
+	case 10:
+		if (cbor_decode_uint64(val, &prot) < 0)
+			return (-1);
+		return (fido_cred_set_prot(cred, (int)prot));
```

Comment:
Is this conversion fine without range checks?

Reply:
I force-pushed a new version that does mimic the robust check from pin.c and turns every negative return value of fido_cred_set_prot into a -1.